test/integration/test-fixtures/image-java-no-main-package/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jenkins:2.60.3
+FROM jenkins/jenkins:2.346.3-slim-jdk17
```

Comment:
Thanks for the update here - was this causing an issue on macOS ARM?

Reply:
Yes, there were no arm images prior to this version, or at least... this version works with our tests and provides arm images :)